docker-compose.yml

```diff
@@ -10,6 +10,6 @@ mysql:
 wordpress:
   build: .
   ports:
-    - "8000:80"
+    - "80:80"
```

Comment:
Please update line 25 in the Vagrantfile to forward port 80 on the VM guest to port 8000 on the host.